src/uwtools/drivers/ww3.py

```diff
@@ -66,7 +66,10 @@ def provisioned_run_directory(self):
         Run directory provisioned with all required content.
         """
         yield self._taskname("provisioned run directory")
-        yield [self.namelist_file(), self.restart_directory()]
+        yield [
+            self.namelist_file(),
+            self.restart_directory(),
+        ]
```

Comment:
Just for consistency with other drivers where we've forced these items onto multiple lines even when they're short enough for one line.